### PR TITLE
update ggH MC and add EWZ xs uncertainty

### PR DESCRIPTION
**Pull request by nanlu06**

### CODE REVIEW

File: tests/hmm/pars.py [hunk 1]
```diff
@@ -400,14 +404,24 @@
...
+    ("ggh_amcPS_TuneCP5down", "2016", "/store/mc/RunIISummer16NanoAODv5/GluGluHToMuMu_M125_TuneCP5down_PSweights_13TeV_amcatnloFXFX_pythia8/**/*.root", True),
+    ("ggh_amcPS_TuneCP5up", "2016", "/store/mc/RunIISummer16NanoAODv5/GluGluHToMuMu_M125_TuneCP5up_PSweights_13TeV_amcatnloFXFX_pythia8/**/*.root", True),
+    ("ggh_powheg", "2016", "/store/mc/RunIISummer16NanoAODv5/GluGlu_HToMuMu_M125_13TeV_powheg_pythia8/**/*.root", True),
+    ("ggh_powhegPS", "2016", "/store/mc/RunIISummer16NanoAODv5/GluGluHToMuMu_M-125_TuneCP5_PSweights_13TeV_powheg_pythia8/**/*.root", True),
+
     ("ggh_amc", "2017", "/store/mc/RunIIFall17NanoAODv5/GluGluHToMuMu_M125_13TeV_amcatnloFXFX_pythia8/**/*.root", True),
     ("ggh_amcPS", "2017", "/store/mc/RunIIFall17NanoAODv5/GluGluHToMuMu_M125_TuneCP5_PSweights_13TeV_amcatnloFXFX_pythia8/**/*.root", True),
-    ("ggh_amcPS", "2018", "/store/mc/RunIIAutumn18NanoAODv5/GluGluHToMuMu_M125_TuneCP5_PSweights_13TeV_amcatnloFXFX_pythia8/**/*.root", True),    
-    ("vbf", "2016", "/store/mc/RunIISummer16NanoAODv5/VBFHToMuMu_M125_TuneCP5_PSweights_13TeV_amcatnlo_pythia8/**/*.root", True),
```
Comment (jpata) [inline tests/hmm/pars.py]: intended to remove vbf 2016 or typo?

Comment (nanlu06) [reply tests/hmm/pars.py]: It's a typo. Sorry, I added it back.

**Comment by jpata** [comment]
Thanks. I added an invitation to you so you can also press the merge "Squash and merge" button yourself when done.
One thing, somehow there are still your commits from the previous PR in this one. I don't want to push too hard on this, but just to keep things clean, when you contribute a fix, I can suggest the following model:

```
#sync to changes
git checkout master
git pull origin master
git checkout -b my_new_fix

... edit files to make a fix ...

git commit
git push --set-upstream origin my_new_fix

... do a PR review, if confident, merge ...

#sync to changes
git checkout master
git pull origin master
```

**Comment by nanlu06** [comment]
Sure, I should follow this model to keep things clean and I accepted the invitation. Thank you!

**Comment by jpata** [comment]
I agree with all changes so far, feel free to merge.

**Pull request merged**